frontend/src/services/ErrorHandler.ts

```diff
@@ -0,0 +1,36 @@
+import {Dispatch} from 'redux'
+import { toast } from 'react-toastify'
+
+import {errorMessage } from '../constants/errorAndSuccessMessages'
+import {userPreferencesRequestFailed} from '../actions/prefAndProfileActions'
+
+export interface IError {
+        field: string
+        message: string
+        statusCode?: number // код статуса ошибки например 401 или 500
+        errorType?: string // тип ошибки например ошибка формы
+}
+
+export interface IErrorHandlerReturn {
```

Comment:
Where does this interface is used?
Probably it's better to use `string[]` instead of union of standalone and array type.

Reply:
in refactored, our ErrorHandler returns an object with 4 fields and it complies with IErrorHandlerReturn